### ISSUE TITLE
Support (-webkit-)text-stroke properties

### ISSUE THREAD

**Issue by MurakamiShinyu** [vivliostyle/vivliostyle.js#1055]
Spec: https://compat.spec.whatwg.org/#text-fill-and-stroking
- 3.4.6.1 Foreground Text Color: the -webkit-text-fill-color property
- 3.4.6.2 Text Stroke Color: the -webkit-text-stroke-color property
- 3.4.6.3 Text Stroke Thickness: the -webkit-text-stroke-width property
- 3.4.6.4 Text Stroke Shorthand: the -webkit-text-stroke property

Test sample: https://gist.githack.com/MurakamiShinyu/d6db72ed59ef000e908e5d6bdcd43f6c/raw/webkit-text-stroke-and-fill-test.html

```html
<style>
:root { font-size: 400%; }
.test1 {
  -webkit-text-stroke-width: 2px;
  -webkit-text-stroke-color: magenta;
}
.test2 {
  -webkit-text-stroke: 2px magenta;
}
.test3 {
  -webkit-text-stroke: 2px magenta;
  -webkit-text-fill-color: yellow;
}
</style>
<p class=test1>Text Stroke Test</p>
<p class=test2>Text Stroke Shorhand Test</p>
<p class=test3>Text Stroke and Fill Test</p>
```

Test with Vivliostyle Viewer:
https://vivliostyle.vercel.app/#src=https://gist.githack.com/MurakamiShinyu/d6db72ed59ef000e908e5d6bdcd43f6c/raw/webkit-text-stroke-and-fill-test.html

**Comment by MurakamiShinyu** [comment]
Test sample, without -webkit- prefix:
https://gist.githack.com/MurakamiShinyu/4ecdfa965ef3ff79f76d3ab92358403e/raw/text-stroke-and-fill-test.html

**Comment by MurakamiShinyu** [comment]
Fixed in #1057.

Test:

- (with -webkit- prefix) https://vivliostyle-git-feat-text-stroke-vivliostyle.vercel.app/#src=https://gist.githack.com/MurakamiShinyu/d6db72ed59ef000e908e5d6bdcd43f6c/raw/webkit-text-stroke-and-fill-test.html

- (without -webkit- prefix) https://vivliostyle-git-feat-text-stroke-vivliostyle.vercel.app/#src=https://gist.githack.com/MurakamiShinyu/4ecdfa965ef3ff79f76d3ab92358403e/raw/text-stroke-and-fill-test.html


Result screenshot:
![Screenshot 2022-12-12 at 12 06 08](https://user-images.githubusercontent.com/3324737/206952214-6381504d-a49f-433f-8426-fb57bd446268.png)
